conf/modules.config

```
@@ -562,7 +574,8 @@ process {
     }
 
     withName: MAPDAMAGE2 {
-        ext.prefix = { "${meta.sample_id}_${meta.library_id}_rescaled" }
+        tag = { "${meta.reference}${meta.sample_id}_${meta.library_id}" }
```

```suggestion
        tag = { "${meta.reference}|${meta.sample_id}_${meta.library_id}" }
```

Might be godo to check this is not missing anywhere else due to copy-pasta